tools/n64tool.c

```diff
@@ -264,7 +275,13 @@ int main(int argc, char *argv[])
 				fprintf(stderr, "ERROR: Invalid size argument; must be at least %d bytes\n\n", MIN_SIZE);
 				return print_usage(argv[0]);
 			}
-
+			if (size % 4 != 0)

```

Comment:
Did you want to also throw in a warning for 64drive wanting size to be a multiple of 512?

Reply:
We could do it, but I'm on the fence because it would be spam for users of any other dev cartridge, and for users of 64drive via both g64drive and UNFLoader. Opinions?

Reply:
I think we should be encouraging developers to create ROMs that are as widely-compatible as possible. A warning would be helpful for people who don’t know the quirks, and could just be ignored by people who don’t care.

Reply:
OK done. But given your concern, I did something more: I made sure n64tool never generates a ROM which is not multiple of 512 bytes, even when the user has not declared explicitly a size (since it's not mandatory anymore). See the followup commit.